### ISSUE TITLE
Feature request: Print style sheet

### ISSUE THREAD

**Issue by joesb** [eddiesigner/weiss-pro#14]
Hello, I'm a Weiss Pro theme user, using it at https://www.agencyradar.io/

Our users are business leaders who may want to print version of articles to use for internal circulation to their board members prior to a business meeting.

 I was wondering if a future release of Weiss Pro might include a good print style sheet?

**Comment by eddiesigner** [comment]
Hi,

This is quite interesting but I have the feeling that this might not be necessary for most of the people that buy this theme so probably I won't include it as part of the theme, however, if it's necessary for your publication then you can do the following:

1. Head to the Ghost Admin and click on the **Code injection** menu.

2. Paste the following styles in the **Site Header** section:

```html
<style>
	@media print {
		header, footer, iframe, .m-article-hero, .m-article-divider, .l-tags, .m-article-share, .l-author, .l-comments, .l-related-articles, .m-newsletter-section, .l-search, .m-alert, .m-share, .m-scroll-top {
			display: none !important;
		}

		.m-article-header {
			display: block !important;
			margin-bottom: 50px !important;
		}

		img {
			width: 100% !important;
			height: auto !important;
		}

		.m-article-content {
			padding-bottom: 0 !important;
		}
	}
</style>
```

3. Paste the following script in the **Site Footer** section:

```html
<script>
const body = document.querySelector('body');

if (
    body.classList.contains('post-template') ||
    body.classList.contains('page-template')
  ) {
	const head = document.head;
    const link = document.createElement('link');

    link.type = 'text/css';
    link.rel = 'stylesheet';
    link.href = 'https://unpkg.com/gutenberg-css@0.6';
    link.media = 'print';

    head.appendChild(link);
}
</script>
```

4. Save our changes.

Hope it helps!

**Comment by joesb** [comment]
Thanks for your comments and suggestions. I've implemented them using a combination of creating a `print.css` file  and making a couple of small updates to the templates (to add the `print.css` and Guttenberg stylesheet).

I decided in the end that, rather than just hiding elements I would create a specific '.do-not-print' class to `display: none` that I could add to whatever markup I need to. E.g. I wanted to show the site logo but hide other parts in the header, hide some of the footer but show the `.m-footer-copyright` content, and so on. A class I could add where I needed to seemed sensible to me.

Thanks for your help.
